parser.y

```diff
@@ -5582,6 +5586,7 @@ UnReservedKeyword:
 |	"DAY"
 |	"DEALLOCATE"
 |	"DO"
+|	"DUMP"
```

Comment:
It's not in the UnReservedKeyword list:
https://dev.mysql.com/doc/refman/8.0/en/keywords.html

It should be NotKeyword?

Reply:
updated all